itests/kit/blockminer.go

```diff
@@ -283,6 +295,18 @@ func (bm *BlockMiner) InjectNulls(rounds abi.ChainEpoch) {
 	atomic.AddInt64(&bm.nextNulls, int64(rounds))
 }
 
+// Pause compels the miner to wait for a signal to restart
+func (bm *BlockMiner) Pause() {
```

Comment:
These helped me avoid races in test and seem generally useful.  Worth checking carefully to make sure I'm not opening up deadlocks.